agents/triage_agent.py

```diff
@@ -201,6 +202,15 @@ def render_prompt(input: InputSchema) -> str:
 
          * Even if the Jira issue provides a direct link to a fix, you need to validate it
          * When no direct link is provided, you must proactively search for fixes - do not give up easily
+         * Try to use upstream_search tool to find out commits related to the issue. The description
```

Comment:
for better readability and consistency, I wouldn't be afraid to nest the following sentences into points, each beginning with `-`